pkg/platform/runtime/download.go

```diff
@@ -121,27 +121,39 @@ func (r *Download) fetchRecipeID() (strfmt.UUID, *failures.Failure) {
 func (r *Download) FetchArtifacts() (*FetchArtifactsResult, *failures.Failure) {
 	result := &FetchArtifactsResult{}
 
-	platProject, fail := model.FetchProjectByName(r.owner, r.projectName)
-	if fail != nil {
-		return nil, fail
-	}
-	r.orgID = platProject.OrganizationID.String()
-	r.private = platProject.Private
+	var commitID *strfmt.UUID
+	var recipeID strfmt.UUID
+	orgID := strfmt.UUID("00000000-0000-0000-0000-000000000000")
+	projectID := strfmt.UUID("00000000-0000-0000-0000-000000000000")
+	var fail *failures.Failure
 
-	recipeID, fail := r.fetchRecipeID()
-	if fail != nil {
-		return nil, fail
-	}
+	if r.owner != "" || r.projectName != "" {
```

Comment:
This looks like it was bugged before. If a commitID is given then it should override the owner/projectName.